code/modules/power/supermatter/supermatter.dm

```diff
@@ -671,6 +702,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		//if there is more pluox and n2 then anything else, we receive no power increase from heat
 		if(power_changes)
 			power = max((removed.temperature * temp_factor / T0C) * gasmix_power_ratio + power, 0)
+			//Passively gain power when antinoblium_multiplier is above 1.
+			power += (antinoblium_multiplier - 1) * ANTINOBLIUM_POWER_GENERATION * removed.temperature / (removed.temperature + T0C + HEAT_PENALTY_THRESHOLD)
```

Comment:
Why both this and the co2 like stuff?

Reply:
It's supposed to be very unstable and hard to manage. Doing just CO2 stuff will basically just make it like CO2, and if I just did this, most of that power would get instantly removed and would be too easy to manage as power could quickly drop down.

Reply:
co2 math is built to kick in hard at high percents, do you see that happening with current antinob production

Reply:
Yeah it can reach absurdly high powers if you somehow manage to maintain that composition.

Reply:
I don't really like the idea of a "super gas". Would prefer something like needing to mix co2 and antinob rather then antinob just being straight up better

Reply:
That is what I am aiming for. I might increase heat penalty considering latest CO2 changes, but making a pure antinob SM would be extremely hard. It becomes more rewarding, but much more harder the higher the composition gets. The extremely low specific heat would make it heat up easily, and one would want to use CO2 to cool it. I'll test with the latest changes to see how it compares.

Reply:
After testing, I don't think there is any way anyone in the entire world could handle a 100% antinoblium SM, even if given infinite time and infinite resources and gases. CO2 now has a heat penalty of 6, so even a small amount could increase thermal output significantly. I think one would want to combine it with N2 if they don't want it to delam.